ios/plistcodec.go

```diff
@@ -56,3 +57,50 @@ func (plistCodec PlistCodec) Decode(r io.Reader) ([]byte, error) {
 	}
 	return payloadBytes, nil
 }
+
+type PlistCodecReadWriter struct {
```

Comment:
I think this is a really nice idea! 
Please add comments explaining why it's needed. 
Also comment public methods please.